[WIP] Add Dual Monitor Support and Add WiFi Icon Changes

Lukas-Batema opened the pull request:
# Description
WIP - Description will be added once complete and ready for review.

Closes #292 

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

notAperson535 commented:
Why has there been like 5 of these PRs?

Lukas-Batema commented:
One wouldn’t deploy, one is outdated, and this one is the current. Not sure about the other two.

yashash1511 commented:
i don't understand the need for line 20 , 60-100 in taskbar/index.js
![image](https://user-images.githubusercontent.com/89068816/171243891-b54e4aa2-e234-4e96-8ea9-4bad15a88edd.png)

Lukas-Batema commented:
> i don't understand the need for line 20 , 60-90 in taskbar/index.js

I did not add most of that...

yashash1511 reviewed (changes requested):
i don't understand the need for line 20 , 60-90 in taskbar/index.js

also line 90-120 should be in components/wifi.js instead of taskbar/index.js so that it can be used in other places like lockscreen, action centre, etc

Lukas-Batema commented:
> i don't understand the need for line 20 , 60-90 in taskbar/index.js
> 
> also line 90-120 should be in components/wifi.js instead of taskbar/index.js so that it can be used in other places like lockscreen, action centre, etc

Will do.

yashash1511 commented:
![image](https://user-images.githubusercontent.com/89068816/171253282-64383f91-e219-4b64-830e-4033626dad56.png)
u missed this part ig

yashash1511 commented:
> I did not add most of that... dunno why it shows it?

i think the reason could be that earlier we used to have that battery logic in taskbar/index.js later i refactored it to battery.js so it could be reused in other places like lockscreen too.

Lukas-Batema commented:
> > I did not add most of that... dunno why it shows it?
> 
> i think the reason could be that earlier we used to have that battery logic in taskbar/index.js later i refactored it to battery.js so it could be reused in other places like lockscreen too. and you started work on wifi before i had refactored it.

Got it, ~~btw you should check your email lol~~

yashash1511 commented:
> Got it, ~btw you should check your email lol~

weird i didn't receive any email wen did u send it can you resend?

Lukas-Batema commented on src/components/Battery.js:
```diff
@@ -4,27 +4,27 @@ import { Icon } from "../utils/general";
 
-      bt.onlevelchange = () => {
-        changebatterystatus(bt);
+      bt.onLevelChange = () => {
```
Oh, shit. My bad. Will fix soon.

Lukas-Batema replied:
It still worked tho.

yashash1511 replied:
yes it still works

yashash1511 replied:
yes

yashash1511 commented:
also the new battery level just gets updated in one of the 3places it is used (lockscreen, actionCenter, taskbar)

![image](https://user-images.githubusercontent.com/89068816/172907354-de88e11c-6434-4954-b445-1e7cc883db3e.png)
![image](https://user-images.githubusercontent.com/89068816/172907304-77797b52-ca20-4c2c-a2d4-3f6e551ff2e4.png)

i think we can fix this by storing the battery percent in redux-store

Lukas-Batema commented:
> also the new battery level just gets updated in one of the 3places it is used (lockscreen, actionCenter, taskbar)
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/89068816/172907354-de88e11c-6434-4954-b445-1e7cc883db3e.png)
> 
> ![image](https://user-images.githubusercontent.com/89068816/172907304-77797b52-ca20-4c2c-a2d4-3f6e551ff2e4.png)
> 
> 
> 
> i think we can fix this by storing the battery percent in src/reducers/settings.js maybe
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/89068816/172909125-84699ad7-78e3-4b33-a0d3-462c25f5ecbf.png)
> 
> we can add a battery option here

Will do.

yashash1511 commented:
how u doin

Lukas-Batema commented:
One wouldn’t deploy, one is outdated, and this one is the current. Not sure
about the other two.

On Tue, May 31, 2022 at 12:31, notAperson535 ***@***.***>
wrote:

> Why has there been like 5 of these PRs?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/blueedgetechno/win11React/pull/488#issuecomment-1142360266>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANI5RBIRXBY6WSGXKCZODFDVMY5GFANCNFSM5XLFU3OA>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>
-- 
Best Wishes,
Lukas Batema (Java, JavaScript, and Front End Web Developer; Moderator on
PedestriaMC Network)
